.pre-commit-config.yaml

```diff
@@ -129,17 +129,17 @@ repos:
       rev: v1.4
       hooks:
           - id: autoflake
-    - repo: https://github.com/twu/skjold
-      rev: v0.2.1
-      hooks:
-          - id: skjold
+    # - repo: https://github.com/twu/skjold
```

Comment:
Why this commented out?

Reply:
I don't want to remove it but with an update causing an issue I wanted to resolve everything else first. This whole update process is dependency hell so it's still a work in progress.